packages/upset/src/components/ElementView/ElementVisualization.tsx

```diff
@@ -1,36 +1,99 @@
-import { Button } from '@mui/material';
+// import { Button } from '@mui/material';
```

Comment:
```suggestion
```

Reply:
Addressed in 9928adf33c7ec8cc57445051fac0db037439fb9d